examples/autoscaler-webhook/Makefile

```
@@ -47,8 +52,12 @@ push: build
 
 # check if hosted on Google Artifact Registry
 gar-check:
-	gcloud container images describe $(autoscaler_webhook_tag)
+	gcloud container images describe $(PROD_REPO)/$(autoscaler_webhook_tag)
 
 #output the tag for this image
 echo-image-tag:
-	@echo $(autoscaler_webhook_tag)
+	@echo $(PROD_REPO)/$(autoscaler_webhook_tag)
+
+# build and push the autoscaler-webhook image with specified tag
+cloud-build:
+	gcloud builds submit --config=../cloudbuild.yaml
```

While working on the autoscaler-webhook image, I encountered the following error:

Step #2 - "build": Already have image: make-docker
Step #2 - "build": cd / && docker build -f /workspace//Dockerfile --tag=us-docker.pkg.dev/agones-mangalpalli/examples/autoscaler-webhook:0.7 .
Step #2 - "build": error checking context: 'no permission to read from '/dev/mem''.
Step #2 - "build": make: *** [Makefile:47: build] Error 1
It seems to be related to permission issues while trying to access '/dev/mem' during the build process. Could you please advise if there's a specific condition we need to add to the build process to avoid this error?

---

I _think_ we'll need something similar to what we have in `make build`, which will be something like:

```suggestion
	cd $(root_path) && gcloud builds submit --config=./examples/autoscaler-webhook/cloudbuild.yaml
```

And then will likely need up update some of the paths to work from `/workspace/examples/autoscaler-webhook/`